l10n_eu_product_adr/wizards/DG_product_handler.py

```diff
@@ -35,7 +36,6 @@ def prepare_DG_data(self):
             }
         }
         """
```

Comment:
I'd say we can add another `self.ensure_one()` here, others the code looks good to me.